### DISCUSSION TITLE
New sensor, what files to provide in the pull request?

### DISCUSSION THREAD

**Discussion by Vincent1964** [arendst/Tasmota#18737] [General]
Hi,

I am developing a new sensor driver for the MAX17043 battery power gauge. Should my pull request contain also "my_user_config.h" and the "i18n.h" includes?

I also found an error in tasmota/tasmota_xx2c_global/xsns_interface.ino (line 459, &Xsns109 is missing a comma). I propose I add this to my pull request.

Kind regards,
Vincent

**Comment by Jason2866** [comment]
The PR should include the changes in `my_user_config.h` commented out and in `i18n.h`

Bug found in existing code is always an extra standalone PR.

**Comment by arendst** [comment]
I'll fix Xsns1019 now.
